maat-court-data-api/src/main/java/gov/uk/courtdata/dces/request/FdcContributionsRequest.java

```diff
@@ -0,0 +1,23 @@
+package gov.uk.courtdata.dces.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.util.Set;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class FdcContributionsRequest {
```

Comment:
It looks like this class is only used by `FdcContributionsMapper` , can this class be removed if not used?